fetch_cluster_logs.bash

```diff
@@ -108,7 +108,7 @@ fi
 
 # download the DC/OS CLI
 if [ ! -f dcos-cli ]; then
-  wget https://downloads.dcos.io/binaries/cli/linux/x86-64/dcos-1.12/dcos --output-document=dcos-cli
+  wget https://downloads.dcos.io/binaries/cli/linux/x86-64/dcos-1.11/dcos --output-document=dcos-cli
```

Comment:
Thanks for the patch.

Why did this ever work?

Reply:
But should we not learn from that; and *not consider the DC/OS CLI retro-compatible*? Because here we learned that it is not?

Reply:
The DC/OS CLI is not considered/promoted as retro-compatible, something that has been made clearer with our new warning when there is a mismatch between the CLI and the DC/OS version of the cluster. However, having a mismatch does not return an error thus inconsistencies such as the one fixed here exist in our codebase.